tests/framework/e2e/etcdctl.go

```diff
@@ -42,7 +42,8 @@ func (ctl *EtcdctlV3) DowngradeEnable(version string) error {
 }
 
 func (ctl *EtcdctlV3) Get(key string, o config.GetOptions) (*clientv3.GetResponse, error) {
-	args := ctl.cmdArgs()
+	resp := clientv3.GetResponse{}
+	args := []string{}
```

Comment:
nit: `var args []string` is recommended in this case. I recall some go linters report warning on this.